taichi/rhi/vulkan/vulkan_device.cpp

```diff
@@ -782,6 +782,7 @@ VulkanCommandList::VulkanCommandList(VulkanDevice *ti_device,
   info.flags = VK_COMMAND_BUFFER_USAGE_SIMULTANEOUS_USE_BIT;
 
   vkBeginCommandBuffer(buffer->buffer, &info);
+  vkCmdResetQueryPool(buffer->buffer, query_pool_->query_pool, 0, 2);
```

Comment:
Nit: IIRC the length of query_pool is always 2 which stands for start time and end time, is this correct understanding? 
If so, would you mind putting this into a comment so that it's easier to understand ;) ? Thanks!

Reply:
Thats correct, Imma work on the profiler system integration soon after the C-API docs so I'll solve this after that. The magic number should be completely removed actually.